CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+   - Added BYOC_MODE build flow. This enables you to inject the certificates into the image at runtime rather than compile time hence enabling you to create generic developer builds.
+- [edge-tool] Added [Edge tool v0.2.0](https://github.com/PelionIoT/mbed-edge/tree/master/edge-tool).
+   - Installed only when build with `BYOC_MODE=ON`.
+   - Converts the development certificate to CBOR configuration object which is then provided to Edge Core as command line argument.
+- [TBC] LmP RPi3/4 support.
+- [TBC] Add support for Hummingboard Solid with booting from SD card.
+- [TBC] Network policy
+   - allows kubelet and kube-router to function without external network. Supports offline cache mode
+- [TBC] UZ - align wks.in file usage with i.mx8
+- [TBC] Change default image file name for MFG tool - `MFGTOOL_FLASH_IMAGE = "console-image-lmp"`.
+- [TBC] Removed meta-arm-autonomy layer.
+- [TBC] add dependency to pc-ble-driver into config, default off
+- Enable `wifi` by default for all targets.
+- Added imx8mmsolidrun meta layer.
+- [TBC] Added support for full image update.
+- [TBC] Prevent duplicate deployment. 
+   - Modifed to check if the commit has been deployed before making the deployment. This will prevent an issue where a previous deploy can get over-written which in turn would break the rollback functionality.
+- [parsec] Upgraded parsec-se-driver to 0.5.0, parsec-service to 0.7.2 and parsec-tool to 0.3.0.
+   - Set the parsec sock directory permission to 0750
+- [TBC] Simplified the partition layout - https://github.com/PelionIoT/meta-mbed-edge/pull/51 
+- [TBC] Generic `mx8mm` support - Instead of using the imx8mmevk target, let's use mx8mm to generalize the support as the current changed should run on all targets. Generalizing thus to the SoC level target `mx8mm`.
+- [TBC] Removed `networkmanager-nmtui` and instead installed `networkmanager-nmcli`.
+- [TBC] use miniuart bt dtoverlay to enable bluetooth
+   - just use standard console and ble settings
+- [TBC] Use standard tty config, fixes ble
+- [TBC] removes golang overrides to use native version 1.15.8 provided by current yocto branch
+- [edge-proxy] Modified `edge-proxy` configuration to add new forwarding address for containers domain. 
+   - Added `containers.local` to the list of known hosts.
+- Updated `info` utility to v2.0.9 and `identity-tool` to v2.0.8.
+- [fluentbit] Reduced the default FluentBit logging level to warning.
+- [mbed-fcce] Upgrades factory-configurator-client-example to v4.9.0.
+- No longer required to specify this `vendor-id=42fa7b48-1a65-43aa-890f-8c704daade54` to unlock the rich node features in Pelion web portal. Portal now reads the Gateway capabilities from gateway's FeatureMgmt LwM2M object 33457 and then enable/disable the UI associated to the feature.
+
+### Bug fixes
+
+- [info] The `info` command must be run with `sudo` on LMP-based boards (UltraZed-EG IOCC and i.MX 8M Mini EVK).
```

Comment:
These need to converted into format whereby we say "Fixed issue whereby info-command required root access on ..."
LMP -> LmP.

Reply:
```suggestion
- [info] Fixed issue whereby info-command required root access on all LmP supported boards.
```